src/tasks/DockerPseudoterminal.ts

```diff
@@ -120,10 +120,37 @@ export class DockerPseudoterminal implements Pseudoterminal {
             }
         );
     }
+
+    public async executeCommandInTerminal(command: string, options?: ExecuteCommandInTerminalOptions): Promise<ExecAsyncOutput> {
+
+        // Output what we're doing, same style as VSCode does for ShellExecution/ProcessExecution
+        this.write(`> ${command} <\r\n\r\n`, DEFAULTBOLD);
+
+        return await execAsync(
+            command,
+            {
+                cwd: this.resolvedDefinition.options?.cwd || options.folder.uri.fsPath,
+                cancellationToken: options.token,
+            },
+            (output: string, err: boolean) => {
+                if (err) {
+                    this.writeErrorLine(output);
+                } else {
+                    this.writeOutputLine(output);
+                }
+            }
+        );
+    }
+
 }
 
-type ExecuteCommandInTerminalOptions = {
+type ExecuteCommandResponseInTerminalOptions = {
     commandResponse: VoidCommandResponse | PromiseCommandResponse<unknown>;
     folder: WorkspaceFolder;
     token?: CancellationToken;
 };
+
+type ExecuteCommandInTerminalOptions = {

```

Comment:
ExecuteCommandResponseInTerminalOptions should extend this type (since it's a superset of this type)

Reply:
:+1: